clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -366,8 +383,16 @@ public void close() {
         }
     }
 
+    // visible for testing
+    void invokeCompletedOffsetCommitCallbacks() {
+        for (OffsetCommitCompletion completion : completedOffsetCommits)
+            completion.invoke();
```

Comment:
Should these be protected by `try`/`catch` blocks? They invoke user code and if an exception is thrown and handled then we'll just reinvoke them.